src/utils/misc.cpp

```diff
@@ -734,47 +734,54 @@ QString Utils::Misc::parseTaskList(const QString &html, bool clickable) {
 
     // set a list item style for todo items
     // using a css class didn't work because the styling seems to affects the sub-items too
-    const auto listTag = QStringLiteral("<li style=\"list-style-type:square\">");
+    const auto taskListTag = QStringLiteral("<li class=\"task-list-item-checkbox\">");
```

Comment:
Isn't the `-checkbox` a little much now?

Reply:
Could be. @pbek what do you think?

Reply:
The smaller (yet readable) the better

Reply:
I'd also turn it around in case we'll think of more ;)
`list-item-task`